### PR TITLE
Capstone: Prize_giving

### PR DESCRIPTION
**Pull request by MPDADDY**
This is a webpage that communicates the oncoming prize-giving as well as the venue where it will be held.
Implemented the changes requested by the previous reviewer.
Implemented changes requested by the second reviewer.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,176 @@
...
+
+- **[Header Section]** 
+Contains a mobile humburger menu inthe mobile version
+- **[parners section]**
+Contains partners logos.
+- **[footer]**
+COntains the meeting logo and the copyright
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Live Demo  - click [TriumphTribute]()
```
Comment (nedjwab) [inline README.md]: [ ] kindly add the link to your deployed website here, so your readme looks more professional.🤓

File: about.html [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+            <img src="images/googlekorea 1.png" alt="googlekorea logo">
+            <img src="images/daumkakaologo 1.png" alt="daumkakaologo">
+            <img src="images/naver 1.png" alt="naver logo">
+          </div>
+        </article>
+      </section>
+    </main>
+    <footer class="aboutfooter">
+      <div class="footer">
+        <a href="index.html">TriumphTribute</a>
```
Comment (nedjwab) [inline about.html]: [ ] Kindly change the footer's color on the About page ( desktop version) to black to match the design provided.🤓
your design 👇:
his

Figma design 👇 :
design

File: about.html [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+          </li>
+          <li>
+            <a class="mynav" href="#">sponser</a>
+          </li>
+          <li>
+            <a class="mynav" href="#">news</a>
+          </li>
+        </ul>
+        <!--desktop menu-->
+        <div class="deskMenu">
```
Comment (nedjwab) [inline about.html:72]: [ ] Kindly add some margin so your navbar matches the one in the Figma design🤓
Your design 👇: 
navhis

Figma design 👇 : 
navd

**Review by nedjwab — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @MPDADDY 
Nice job, you are almost there.



## (Highlights) Good Points: :
- Good implementation of some of the project requirements  ✔
- GitHub flow was correctly used ✔.

## (Changes Required) Aspects to improve: ♻️
You did great implementing some of the project requirements, but you still have some issues: 
check the comments below 
 👇👇👇

 
## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,686 @@
...
+.aboutFooter {
+  background-color: #3e3e3e;
+  padding: 3rem 1rem;
+  color: var(--white);
+}
+
+.white {
+  color: var(--white) !important;
+}
+
```
Comment (cynthiainga) [inline styles.css:523]: - [ ] Good job implementing the requested changes for About's footer but in the mobile menu now the text is invisible. The background is white and the text is in white also, that is why we can't see the text, the text should be in black color. Kindly fix that 🙏🏾 

Screenshot 2023-08-01 at 15 44 34

File: README.md [hunk 5]
```diff
@@ -1,2 +1,176 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (cynthiainga) [inline README.md]: - [ ] The first reviewer forgot to mention it but you should fix the broken MIT link, having broken links in the README is not very professional 🙃 🙃 

Screenshot 2023-08-01 at 16 16 07

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @MPDADDY, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights

- [ ] No linters errors ✔️ 
- [ ] Correct GitHub flow ✔️ 
- [ ] Good design ✔️ 

 ## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇🏾 👇🏾 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @MPDADDY 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**